geojson_pydantic/geometries.py

```diff
@@ -65,6 +65,16 @@ def check_coordinates(cls, polygon):
 
         return polygon
 
+    @classmethod
+    def from_bounds(cls, bounds: BBox) -> "Polygon":
```

Comment:
I think the signature should be `def from_bounds(cls, xmin, ymin, xmax, ymax)` for three reasons:
1. It aligns with the shapely implementation (https://github.com/Toblerity/Shapely/blob/0a29437447567e7307d4265335b628423c9c286f/shapely/geometry/polygon.py#L381), and we're already doing other things like implementing `__geo_interface__` to be interoperable with shapely.
2. It's explicit in the function signature the order of coordinates within the specified bounds.
3. Star notation is nice - `_GeometryBase.from_bounds(*bounds)`

Reply:
Agreed 🙏